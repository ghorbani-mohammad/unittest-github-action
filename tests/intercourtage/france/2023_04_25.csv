

"GRAIN/SHP/FR","FR Port Activity","","","","",""


"Export estimation:","","WHEAT","","<0#WHT-FREXP>","",""


"<GRAINS/FREXP>","","","MAIZEincl.WAXY","<0#MAZ-FREXP>","",""


"<0#GRAINS-FREXP>","","","BARLEYincl.MALT","<0#BLY-FREXP","",""


"","","DURUM","","<O#DUR-FREXP>","",""


"SHIP","PRODUCTS","TONNES","DESTINATION","OPERATOR","STAT","DAY"
"ROUEN","","","","","",""
"BUNUM INFINITY","MALT","8 000","BRAZIL","SOUFFLET","L","26Apr"
"THOR FORTUNE","WHEAT","40 000","MOROCCO","SOUFFLET","L","26Apr"
"VITA FUTURE","BARLEY","40 000","SINGAPORE","VITERRA","A","26Apr"
"VOGUE SOPHIE","WHEAT","21 000","AFRICA","LECUREUR","A","27Apr"
"NORDIC BC MUNICH","MALT","8 000","SOUTH AFRICA","SOUFFLET","A","27Apr"
"ADNAN","MALT.BI","10 000","TURKEY","SOUFFLET","A","27Apr"
"BOCS SOFI","MALT","2 900","CAMEROON","SACOFRINA","A","28Apr"
"SOND YUNG","BARLEY","50 000","CHINA","BEUZELIN","A","30Apr"
"VITA FUTURE","BARLEY","20 000","SINGAPORE","VITERRA","A","30Apr"
"CAEN","","","","","",""
"LA PALLICE","NONE","LOADING","NONE","EXPECTED","","28Apr"
"ATLANTIC BORGE","GRAINS","5 000","WEST INDIES","SOUFFLET","L","27Apr"
"MANISA STAR","WHEAT","6 000","PORTUGAL","SOUFFLET","L","27Apr"
"ICY BAY","WHEAT","30 000","ANGOLA","SOUFFLET","A","28Apr"
"ZING ING XI HI","WHEAT","42 000","SENEGAL","EURAFRIQUE","A","02May"
"SOND YUNG","BARLEY","8 000","CHINA","BEUZELIN","A","06May"
"BOCS BREMEN","MAIZE","3 000","CAMEROON","CAVAC","A","12May"
"TONNAY/ROCHEFORT","","","","","",""
"FRI TIDE","MAIZE","3 300","UK","ENTENTE","A","29Apr"
"NANTES","","","","","",""
"HSL HONOLULU","WHEAT","10 000","CHINA","BUNGE","L","28Apr"
"MONTOIR","","","","","",""
"ST NAZAIRE","NONE","LOADING","NONE","EXPECTED","","28Apr"
"BORDEAUX","NONE","LOADING","NONE","EXPECTED","","28Apr"
"BLAYE","NONE","LOADING","NONE","EXPECTED","","28Apr"
"BAYONNE","NONE","LOADING","NONE","EXPECTED","","28Apr"
"RICS SPIRIT","WAXY","3 500","NETHERLANDS","MAISADOUR","L","27Apr"
"EIKBORG","WAXY","3 500","GERMANY","MAISADOUR","L","28Apr"
"PORT LA NOUVELLE","NONE","LOADING","NONE","EXPECTED","","28Apr"
"DUNKERQUE","NONE","LOADING","NONE","EXPECTED","","27Apr"
"TRADE","WHEAT","20 000","CHINA","SOUFFLET","L","27Apr"
"MAINE SOLEIL","BARLEY","20 000","CHINA","SOUFFLET","A","28Apr"
"VITA FUTURE","BARLEY","20 000","SINGAPORE","SOUFFLET","A","30Apr"

