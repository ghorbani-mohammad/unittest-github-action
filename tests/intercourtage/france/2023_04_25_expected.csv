report_date,contributor_lineup_reference_id,commodity_name,commodity_specification_codes,volume,unit,origin_code,destination_code,expected_cargo_loading_date,cargo_loading_date,cargo_discharged_date,cargo_status,carrier_id,carrier_name,carrier_status,carrier_dead_weight,export_location,import_location,export_terminal,import_terminal,eta,etb,etd,ata,atb,atd,agent,charterer,shipper,operator,comments
None,None,MALT,None,8000.0,MT,None,None,2023-04-26T00:00:00+00:00,None,None,None,None,BUNUM INFINITY,None,None,None,BRAZIL,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,WHEAT,None,40000.0,MT,None,None,2023-04-26T00:00:00+00:00,None,None,None,None,THOR FORTUNE,None,None,None,MOROCCO,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,BARLEY,None,40000.0,MT,None,None,2023-04-26T00:00:00+00:00,None,None,None,None,VITA FUTURE,None,None,None,SINGAPORE,None,None,None,None,None,None,None,None,None,None,None,VITERRA,None
None,None,WHEAT,None,21000.0,MT,None,None,2023-04-27T00:00:00+00:00,None,None,None,None,VOGUE SOPHIE,None,None,None,AFRICA,None,None,None,None,None,None,None,None,None,None,None,LECUREUR,None
None,None,MALT,None,8000.0,MT,None,None,2023-04-27T00:00:00+00:00,None,None,None,None,NORDIC BC MUNICH,None,None,None,SOUTH AFRICA,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,MALT.BI,None,10000.0,MT,None,None,2023-04-27T00:00:00+00:00,None,None,None,None,ADNAN,None,None,None,TURKEY,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,MALT,None,2900.0,MT,None,None,2023-04-28T00:00:00+00:00,None,None,None,None,BOCS SOFI,None,None,None,CAMEROON,None,None,None,None,None,None,None,None,None,None,None,SACOFRINA,None
None,None,BARLEY,None,50000.0,MT,None,None,2023-04-30T00:00:00+00:00,None,None,None,None,SOND YUNG,None,None,None,CHINA,None,None,None,None,None,None,None,None,None,None,None,BEUZELIN,None
None,None,BARLEY,None,20000.0,MT,None,None,2023-04-30T00:00:00+00:00,None,None,None,None,VITA FUTURE,None,None,None,SINGAPORE,None,None,None,None,None,None,None,None,None,None,None,VITERRA,None
None,None,GRAINS,None,5000.0,MT,None,None,2023-04-27T00:00:00+00:00,None,None,None,None,ATLANTIC BORGE,None,None,None,WEST INDIES,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,WHEAT,None,6000.0,MT,None,None,2023-04-27T00:00:00+00:00,None,None,None,None,MANISA STAR,None,None,None,PORTUGAL,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,WHEAT,None,30000.0,MT,None,None,2023-04-28T00:00:00+00:00,None,None,None,None,ICY BAY,None,None,None,ANGOLA,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,WHEAT,None,42000.0,MT,None,None,2023-05-02T00:00:00+00:00,None,None,None,None,ZING ING XI HI,None,None,None,SENEGAL,None,None,None,None,None,None,None,None,None,None,None,EURAFRIQUE,None
None,None,BARLEY,None,8000.0,MT,None,None,2023-05-06T00:00:00+00:00,None,None,None,None,SOND YUNG,None,None,None,CHINA,None,None,None,None,None,None,None,None,None,None,None,BEUZELIN,None
None,None,MAIZE,None,3000.0,MT,None,None,2023-05-12T00:00:00+00:00,None,None,None,None,BOCS BREMEN,None,None,None,CAMEROON,None,None,None,None,None,None,None,None,None,None,None,CAVAC,None
None,None,MAIZE,None,3300.0,MT,None,None,2023-04-29T00:00:00+00:00,None,None,None,None,FRI TIDE,None,None,None,UK,None,None,None,None,None,None,None,None,None,None,None,ENTENTE,None
None,None,WHEAT,None,10000.0,MT,None,None,2023-04-28T00:00:00+00:00,None,None,None,None,HSL HONOLULU,None,None,None,CHINA,None,None,None,None,None,None,None,None,None,None,None,BUNGE,None
None,None,WAXY,None,3500.0,MT,None,None,2023-04-27T00:00:00+00:00,None,None,None,None,RICS SPIRIT,None,None,None,NETHERLANDS,None,None,None,None,None,None,None,None,None,None,None,MAISADOUR,None
None,None,WAXY,None,3500.0,MT,None,None,2023-04-28T00:00:00+00:00,None,None,None,None,EIKBORG,None,None,None,GERMANY,None,None,None,None,None,None,None,None,None,None,None,MAISADOUR,None
None,None,WHEAT,None,20000.0,MT,None,None,2023-04-27T00:00:00+00:00,None,None,None,None,TRADE,None,None,None,CHINA,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,BARLEY,None,20000.0,MT,None,None,2023-04-28T00:00:00+00:00,None,None,None,None,MAINE SOLEIL,None,None,None,CHINA,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
None,None,BARLEY,None,20000.0,MT,None,None,2023-04-30T00:00:00+00:00,None,None,None,None,VITA FUTURE,None,None,None,SINGAPORE,None,None,None,None,None,None,None,None,None,None,None,SOUFFLET,None
